package.json

```diff
@@ -28,6 +30,8 @@
     "express": "^4.16.3",
     "moment": "^2.22.2",
     "nuxt": "^2.2.0",
+    "nuxt-material-design-icons": "^1.0.4",
```

Comment:
Hast du es mal wie hier beschrieben versucht? 
https://vuetifyjs.com/en/framework/icons#introduction
mdi installieren
In plugins/vuetify.js `iconfont: 'mdi'`
In assets/style/app.scss `@require 'mdi/font/css/materialdesignicons.css'`

Das braucht keine weiteren imports und man kann `<v-icon>` benutzen.